lib/registry.js

```diff
@@ -118,7 +119,17 @@ util.inherits(RegistryItem, Object);
 Object.freeze(RegistryItem);
 Object.freeze(RegistryItem.prototype);
 
-
+/**
+ * Get the path to system's reg.exe. Useful when another reg.exe is added to the PATH
+ * Implemented only for Windows
+ */
+function getRegExePath() {
+    if (process.platform === 'win32') {
+        return path.join(process.env.windir, 'system32', 'reg.exe');
```

Comment:
not really. child_process.spawn() should handle cases where the executable is missing and reflect the error using #25 .